bokehjs/src/coffee/models/annotations/arrow.coffee

```diff
@@ -12,8 +12,8 @@ class ArrowView extends Annotation.View
     if not @mget('source')?
       this.mset('source', new ColumnDataSource.Model())
     @canvas = @plot_model.get('canvas')
-    @xmapper = @plot_view.frame.get('x_mappers')[@mget("x_range_name")]
-    @ymapper = @plot_view.frame.get('y_mappers')[@mget("y_range_name")]
+    @xmapper = @plot_view.frame.x_mappers[@mget("x_range_name")]
+    @ymapper = @plot_view.frame.y_mappers[@mget("y_range_name")]
```

Comment:
This will be done in another PR(s). Originally I wanted to do it all at once, but then bokehjs was failing a lot. First computed properties, then `m{get,set}` and then finally `get,set`.